README.md

```diff
@@ -34,10 +34,10 @@ function onShutdown () {
   console.log('cleanup finished, server is shutting down');
 }
 
-function healthcheck () {
+function healthCheck () {
```

Comment:
I am not sure if I am following - there was a PR merged that changed it to `healthcheck`, how come the sudden change of heart?